op_robot_tests/tests_files/brokers/openprocurement_client.robot

```diff
@@ -11,8 +11,8 @@ Library  openprocurement_client.utils
   Log Many  ${USERS.users['${username}'].id_map}
   ${status}=  Run Keyword And Return Status  Dictionary Should Contain Key  ${USERS.users['${username}'].id_map}  ${tender_uaid}
   Run Keyword And Return If  ${status}  Get From Dictionary  ${USERS.users['${username}'].id_map}  ${tender_uaid}
-  Call Method  ${USERS.users['${username}'].client}  get_tenders
-  ${tender_id}=  Wait Until Keyword Succeeds  5x  30 sec  get_tender_id_by_uaid  ${tender_uaid}  ${USERS.users['${username}'].client}  id_field=auctionID
+  Call Method  ${USERS.users['${username}'].public_client}  get_tenders
```

Comment:
Do we really need this line here? The value returned by `get_tenders` is not stored anywhere.